src/globalPrototype.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-extend-native */
```

Comment:
I would agree with eslint here, its better to have it as separate functions instead of extending array.